website/docs/components/card/partials/guidelines/guidelines.md

```diff
@@ -8,29 +8,45 @@ Use a card container to help build more consistently styled cards.
 
 ### Static cards
 
-Static cards may have interactive elements within them but are not actionable or interactive on their own.
+Static cards may contain interactive elements but must not be actionable or interactive on their own. They should not have any shadows applied to them, so that they can be differentiated from an interactive card.
+
+![A static card titled "Billing summary" with an info notification and a link.](/assets/components/card/card-static-flat-looking.png)
 
 ### Interactive cards
 
-Interactive cards are actionable (e.g., it can be hovered, clicked, focused, etc.)
+Interactive cards are actionable, e.g., they can be hovered over, clicked, or focused on. If a card is interactive, it should not include nested interactive elements within it.
```

Comment:
[nit] The below "dont" example does a more thorough job of explaining nested interactivity, so I don't think we need it here. 

```suggestion
Interactive cards are actionable, e.g., they can be hovered over, clicked, or focused on. 
```